.github/ISSUE_TEMPLATE/task.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Task
+about: Define precise the subtask of the feature or single independent task
+title: ''
+labels: task
+assignees: ''
+
+---
+
+**Summary**
+Clear description or/and business justification of what the task is about.
+
+**Details**
+If applicapable, detailed information regarding things to be considered, requirements etc.
```

Comment:
Misspelled. Should be "applicable".

Reply:
@Crystalzord Good call. In that case anyway it's also misspelled on main oscm repo